README.md

```diff
@@ -19,118 +13,172 @@
   - [🛠 Built With](#built-with)
     - [Tech Stack](#tech-stack)
     - [Key Features](#key-features)
-
+  - [🚀 Live Demo](#live-demo)
 - [💻 Getting Started](#getting-started)
   - [Setup](#setup)
   - [Prerequisites](#prerequisites)
   - [Install](#install)
-  
+  - [Usage](#usage)
   - [Run tests](#run-tests)
-  
+  - [Deployment](#triangular_flag_on_post-deployment)
 - [👥 Authors](#authors)
-
+- [🔭 Future Features](#future-features)
 - [🤝 Contributing](#contributing)
 - [⭐️ Show your support](#support)
 - [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
-
-
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 finish mobile version <a name="about-project"></a>
+# 📖 My Portfolio <a name="about-project"></a>
 
+> This is my Portfolio I created it using HTML, CSS and JavaScript.
 
+## 🛠 Built With <a name="built-with"></a> 
 
-**mobile first** is an introductory project.
 
-## 🛠 Built With <a name="built-with"></a>
+1- HTML.
+2- CSS.
+3- Liners.
 
 ### Tech Stack <a name="tech-stack"></a>
-css and html
-
 
+> Describe the tech stack and include only the relevant sections that apply to your project.
 
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">HTML</a></li>
+    <li><a href="https://reactjs.org/">CSS</a></li>
+    <li><a href="https://reactjs.org/">JavaScript</a></li>
+  </ul>
+</details>
 
 <!-- Features -->
 
 ### Key Features <a name="key-features"></a>
 
+> Describe between 1-3 key features of the application.
```

Comment:
- [ ] [Optional] Kindly remove all the text describing what you need to do in each heading of your readme file
i.e
> Describe between 1-3 key features of the application.
> Describe the tech stack and include only the relevant sections that apply to your project.
> Describe how a new developer could make use of your project.

Kindly remove all this. 👍